android/app/src/main/java/com/reactnativenavigation/activities/BaseReactActivity.java

```diff
@@ -355,6 +389,20 @@ public void invokeDefaultOnBackPressed() {
         super.onBackPressed();
     }
 
+    protected void setupDrawer(Screen screen, Drawer drawer, int drawerFrameId, int drawerLayoutId) {
```

Comment:
We also don't have to pass the id's as params to `setupDrawer` since in both layout files we use the same ids for the views. `findViewById` will search for these views in the layout which was passed to `setContentView`. No biggie, I'll merge and make that minor change later.
@jskuby Thanks a lot for your work!